yt_dlp/extractor/tvp.py

```diff
@@ -560,12 +560,23 @@ class TVPVODVideoIE(TVPVODBaseIE):
             'thumbnail': 're:https?://.+',
         },
         'params': {'skip_download': 'm3u8'},
+    }, {
+        'url': 'https://vod.tvp.pl/live,1/tvp-world,399731',
+        'info_dict': {
+            'id': '399731',
+            'ext': 'mp4',
+            'title': r're:TVP WORLD(?: \d{4}-\d{2}-\d{2} \d{2}:\d{2})?',
```

Comment:
Why is the date optional?
```suggestion
            'title': r're:TVP WORLD \d{4}-\d{2}-\d{2} \d{2}:\d{2}',
```

Reply:
i randomly got just "TVP WORLD" without a date during testing

Reply:
Hmm, very weird. The test should fail in that case, though. We should be fixing the code instead of rigging the test